app/controllers/merchant_invoices_controller.rb

```diff
@@ -0,0 +1,12 @@
+class MerchantInvoicesController < ApplicationController
```

Comment:
Great to see MerchantInvoicesController for routing those paths to the correct actions